src/useq/_mda_sequence.py

```diff
@@ -388,6 +394,7 @@ def iter_sequence(
     base_event_kwargs: MDAEventDict | None = None,
     event_kwarg_overrides: MDAEventDict | None = None,
     position_offsets: PositionDict | None = None,
+    shutter_plan: ShutterOpenAxes | None = None,
```

Comment:
already add stuff to our nice and trimmed down signature? ;)

this isn't necessary.  see note below